acme/acme/challenges_test.py

```diff
@@ -359,11 +359,14 @@ def test_simple_verify(self, mock_verify_cert):
             cert=mock.sentinel.cert))
         mock_verify_cert.assert_called_once_with(self.msg, mock.sentinel.cert)
 
-    def test_simple_verify_false_on_probe_error(self):
+    @mock.patch('acme.challenges.socket.gethostbyname')
+    def test_simple_verify_false_on_probe_error(self, mock_gethostbyname):
         chall = mock.Mock()
         chall.probe_cert.side_effect = errors.Error
+
+        mock_gethostbyname.return_value = '127.0.0.2'
         self.assertFalse(self.msg.simple_verify(
-            self.chall, self.domain, self.key.public_key()))
+            self.chall, self.domain, self.key.public_key(), host="127.0.0.1"))
```

Comment:
`probe_cert` is the only function in the entire module that does `gethostbyname` and it should already be mocked in `chall.probe_cert.side_effect = errors.Error` line. If it doesn't work then the whole test suite seems to be broken. So you're not fixing the root cause.

Indeed, we call `self.msg.simple_verify(self.chall, ...)` instead `self.msg.simple_verify(chall, ...)`.